sql/handler.cc

```diff
@@ -6762,6 +6762,13 @@ int handler::binlog_log_row(TABLE *table,
   THD *thd= table->in_use;
   DBUG_ENTER("binlog_log_row");
 
+#ifdef WITH_WSREP
```

Comment:
Similar piece of code was removed from this method in 91ab42a823b244a3d4b051ab79701b7a552f274a. To my understanding, the intention was to introduce `handler::prepare_for_row_logging()` which could be called to precompute whether row logging should be used or not. Maybe this call is either missing from the codepath when `wsrep_osu_method=RSU` or it computes the wrong result?

Reply:
Moved condition to more correct place(s).